scripts/build_load_data.py

```diff
@@ -0,0 +1,442 @@
+        w = pd.Timedelta(weeks=1)
+        
+        load.loc[start:stop, 'MK'] = load.loc[start+w:stop+w, 'MK'].values
+       
+        # To fill two inconsistent values 
+        date_1 = pd.Timestamp('2018-09-19 23:00')
+        date_2 = pd.Timestamp('2018-12-13 09:00')
+        w = pd.Timedelta(hours=1)
+        
+        load.loc[date_1, 'MK'] = load.loc[date_1+w, 'MK']
+        load.loc[date_2, 'MK'] = load.loc[date_2+w, 'MK']
+        
+        load['MK'] = load['MK'].interpolate()
+      
+        # Kosovo (KV) and Albania (AL) do not exist in the data set
+        # Kosovo (KV) gets the same load curve as Serbia (RS)
+        # scale parameter selected by energy consumption ratio from IEA Data browser for the year 2017
+        # https://www.iea.org/data-and-statistics?country=KOSOVO&fuel=Electricity%20and%20heat&indicator=Electricity%20final%20consumption
+        load['KV'] = load['RS'] * (5. / 33.)
+        # Albania (AL) gets the same load curve as Macedonia (MK)
+        # scale parameter selected by energy consumption ratio from IEA Data browser for the year 2017
+        # https://www.iea.org/data-and-statistics?country=ALBANIA&fuel=Electricity%20and%20heat&indicator=Electricity%20final%20consumption
+        load['AL'] = load['MK'] * (6.0 / 7.0)
+        
+        logger.info(f"Missing load data are interpolating linearly and adjusted manual.")
+        logger.info(f"Several load data are missing for 'GR'. Manual processing not possible. Copying the data from another source would be one possibility.")
+
+    return load
+
+
+def nan_statistics(df):
+    def max_consecutive_nans(ds):
+        return (ds.isnull().astype(int)
+                  .groupby(ds.notnull().astype(int).cumsum())
+                  .sum().max())
+    consecutive = df.apply(max_consecutive_nans)
+    total = df.isnull().sum()
+    max_total_per_month = df.isnull().resample('m').sum().max()
+    return pd.concat([total, consecutive, max_total_per_month],
+                 keys=['total', 'consecutive', 'max_total_per_month'], axis=1)
+
+
+if __name__ == "__main__":
+
+    if 'snakemake' not in globals():
+        from _helpers import mock_snakemake
+        snakemake = mock_snakemake('build_load_data')
+        rootpath = '..'
+    else:
+        rootpath = '.'
+    
+    configure_logging(snakemake)
+    
+    url = snakemake.config['load']['url']
+    
+    # Save location
+    to_fn = Path(f"{rootpath}/data/time_series_60min_singleindex.csv")
+
+    logger.info(f"Downloading load data from '{url}'.")
+    
+    progress_retrieve(url, to_fn)
+
+    logger.info(f"Raw load data available at '{to_fn}'.")
+    
+    opsd_load = (load_timeseries_opsd(years = slice(*pd.date_range(freq='y', **snakemake.config['snapshots'])[[0,-1]].year.astype(str)),
+                                 fn=to_fn,
+                                 countries = snakemake.config['countries'],
+                                 source = snakemake.config['load']['source']))
```

Comment:
```suggestion
    opsd_load = (load_timeseries_opsd(years = slice(*pd.date_range(freq='y', **snakemake.config['snapshots'])[[0,-1]].year.astype(str)),
                                 fn=url,
                                 countries = snakemake.config['countries'],
                                 source = snakemake.config['load']['source']))
```
Directly reading from url is fine, as we do not really need the raw data

Reply:
OK, I have also thought that it might be useful to separate the loading and processing of load data. Then we could control the automatic loading of the data like in the data bundle. Maybe this is a bit too much work just for the load data. Maybe it makes sense to integrate everything into the load data bundle function.

Reply:
I think it's fine to do it here. Its good to have as many things as possible outside of the databundle. It causes extra work to maintain it properly. The more is retrieved automatically, the better.